app/services/i_scream_service.rb

```diff
@@ -30,6 +29,7 @@ def get_games_by_keywords(keywords)
   end
 
   def json_parse(response)
+    return 404 if response.status != 200
```

Comment:
404!

Reply:
Nice. I noticed that error popped up when I was playing around with the site